tls/s2n_cipher_suites.c

```diff
@@ -1074,6 +1074,33 @@ int s2n_cipher_suites_cleanup(void)
     return 0;
 }
 
+S2N_RESULT s2n_iana_to_cipher_suite(const uint8_t iana[], struct s2n_cipher_suite **cipher_suite)
```

Comment:
I would prefer the name to be prefixed with the module name so it's easier to find

```suggestion
S2N_RESULT s2n_cipher_suite_from_iana(const uint8_t iana[], struct s2n_cipher_suite **cipher_suite)
```